CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2022-06-08
+| What's new | |
+| :--------- | :-: | 
+| Schema checks now provides warning messages when the operation cardinality has exceeded the tier limits rather than just silently ignore them. For details, see [docs](https://www.apollographql.com/docs/studio/schema-checks). | ![list of checks](https://user-images.githubusercontent.com/639025/171964823-fb2762b3-b793-40df-bce3-369367bd4626.png) |
```

Comment:
Doesn't look like we have these limits in the docs yet.  What do you think about getting the docs modified first?  I can help with those changes.

Reply:
Yes, I will need help. Apparently I'm not good with the words around checks, cardinality, or how to soften the message to the customer. This is what I got so far 😞 https://github.com/apollographql/docs/pull/135